Components/alerts/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
As per Smile Gupta PR review, you can add a name in the meta tag.

Reply:
Yes, I shall add it 👍🏽

Reply:
Added desc and author meta tags.